src/bluecoins_cli/adb.py

```diff
@@ -0,0 +1,70 @@
+import datetime
+import subprocess
+
+from adbutils import adb  # type: ignore
+from adbutils._device import AdbDevice  # type: ignore
+
+
+class Device:
+    def __init__(self, device: AdbDevice) -> None:
+        self.device = device
+
+        self.APP_ID = 'com.rammigsoftware.bluecoins'
+
+        current_time = datetime.datetime.now().strftime("%Y-%m-%d_%H:%M:%S")
+        self.DB = f"bluecoins-{current_time}"
```

Comment:
Database name should be generated at the moment of its creation.

Reply:
I did this, but not sure in the way of resolve.

Reply:
@droserasprout